openslides/core/rest_api.py

```diff
@@ -0,0 +1,21 @@
+from openslides.utils.rest_api import RESTElement
+
+from . import views
+
+
+class CustomSlideRESTElement(RESTElement):
+    viewset = views.CustomSlideViewSet
+
+
+class ProjectorRestElement(RESTElement):
+    viewset = views.ProjectorViewSet
+
+
+class TagRestElement(RESTElement):
+    viewset = views.TagViewSet
+
+
+class ConfigRestElement(RESTElement):
+    viewset = views.ConfigViewSet
+    router_prefix = 'config'
+    app_label = 'core'
```

Comment:
You may initiate the `RESTElement` in apps.py not by importing the class but by calling it and use a kind of different implementation (also using a customized __init__()).